packages/in_app_purchase/example/android/app/src/test/java/io/flutter/plugins/inapppurchase/InAppPurchasePluginTest.java

```diff
@@ -57,25 +57,26 @@
 import org.mockito.Spy;
 
 public class InAppPurchasePluginTest {
```

Comment:
nit: This test file should probably be renamed too, since it's really testing MethodCallHandler now.